app/assets/stylesheets/landing.sass

```diff
@@ -0,0 +1,2 @@
+.relative
```

Comment:
I believe bootstrap has a `.position-relative` class that does this already.

Reply:
Sure does. I just grepped in the `app/assets/stylesheets` directory. I didn't check the gems or anywhere else.

Reply:
Since we use the custom paid theme for bootstrap we have it included minified somewhere.  Just be aware that it's still version 4.0.0.